adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/model/Position.java

```diff
@@ -0,0 +1,16 @@
+package com.microservicesteam.adele.ticketmaster.model;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface Position {
+    int sectorId();
```

Comment:
Nitpicking here: if `bookingIds` are `long`, these ids should be as well for consistency.